index.d.ts

```diff
@@ -39,7 +44,15 @@ export interface Hooks {
 /**
 Options are the same as `window.fetch`, with some exceptions.
 */
-interface KyOptions extends RequestInit {
+export interface Options extends RequestInit {
+	/**
+	HTTP request method.
+	*/
+	method?: LiteralUnion<
+		'get' | 'head' | 'post' | 'put' | 'delete' | 'connect' | 'options' | 'trace' | 'patch',
```

Comment:
I prefer the ordering to be sorted by common usage: get, post, put, delete, patch, head, connect, options, trace.

Also put them on separate lines.